parameter/ParameterMgr.cpp

```diff
@@ -1820,6 +1822,37 @@ CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::showMappingCommandP
     return CCommandHandler::ESucceeded;
 }
 
+CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::listMappingsCommandProcess(
+    const IRemoteCommand &remoteCommand, string &strResult)
+{
+    CElementLocator elementLocator(getSystemClass(), false);
+
+    CElement *pLocatedElement = nullptr;
+
+    if (!elementLocator.locate(remoteCommand.getArgument(0), &pLocatedElement, strResult)) {
+        return CCommandHandler::EFailed;
+    }
+
+    if (!pLocatedElement) {
+        // List from root folder
+        pLocatedElement = getSystemClass();
+    }
+
+    vector<string> paths;
+    pLocatedElement->getSubpaths(paths);
+
+    for (string path : paths) {
+        string mapping;
+        if (getParameterMapping(path, mapping)) {
```

Comment:
Can you check if you can dump only the terminal element. Today you are listing the full list of element. So to many element without control are listed. See under:

/Audio/REALTEK/SOUND_CARD/OUTPUT/HEADPHONE/ANALOG/VOLUME => Card:bytcrrt5640, Debug:, Amend1:HP, Volume:'%1 Playback Volume', Volume:HP Playback Volume
/Audio/REALTEK/SOUND_CARD/OUTPUT/HEADPHONE/ANALOG/VOLUME/0 => Card:bytcrrt5640, Debug:, Amend1:HP, Volume:'%1 Playback Volume', Volume:'%1 Playback Volume'

/Audio/REALTEK/SOUND_CARD/OUTPUT/EARPIECE/ANALOG => Card:bytcrrt5640, Debug:
/Audio/REALTEK/SOUND_CARD/OUTPUT/EARPIECE/ANALOG/SWITCH => Card:bytcrrt5640, Debug:, Control:'Mono Playback Switch', Control:Mono Playback Switch